src/frontend/src/components/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+import { useLocation, useRouteMatch, Link as RouterLink } from 'react-router-dom';
+import { Tab, Tabs } from '@mui/material';
+
+interface TabProps {
+  setTab: (value: number) => void;
+  tabUrlValues: string[]; // Values that go in the URL depending on the tab value, example /projects/0.0.0/scope
```

Comment:
shouldnt this be  { tabUrlValues: string; tabNames: string }[]

Reply:
sure